Backend/Django/Django/settings.py

```diff
@@ -32,7 +32,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['206.189.188.197']
+ALLOWED_HOSTS = ['206.189.188.197', 'localhost']
```

Comment:
If you can just remove the localhost. Rather keep it for local development. It will be published soon and having this is no point for the develop branch